openapi.yml

```diff
@@ -392,6 +394,24 @@ paths:
                     type: string
                     description: Health status of the service
                     example: OK
+  /batch/mark_thread_for_deletion:
+    patch:
+      tags:
+        - Batch
+      summary: patch for thread marked_for_deletion
```

Comment:
Is this `mark_threads_for_deletion` plural? Admittedly `/batch` does imply this so maybe being pedantic.

Reply:
A wee bit pedantic. Plural is always a bit of a odd one with API design, with many people going different ways. For me I find it more explicit seeing the threads(s) as i instantly know without any extra information, but yes you could argue batch with Thread means that as well. Providing a list or dict with an end point makes the plural bit even more fun

Reply:
I did have to look at the code to ultimately work out it wasn't acting on a single thread. Both the API and view method do suggest it's acting on a single resource.